### ISSUE TITLE
2024-12-24, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#30]
# Today's Arxiv Papers

(1) [ChatGarment: Garment Estimation, Generation and Editing via Large Language Models](http://arxiv.org/abs/2412.17811v1), 2024-12-23

(2) [ResearchTown: Simulator of Human Research Community](http://arxiv.org/abs/2412.17767v1), 2024-12-23

(3) [In Case You Missed It: ARC 'Challenge' Is Not That Challenging](http://arxiv.org/abs/2412.17758v1), 2024-12-23

(4) [ADC: Enhancing Function Calling Via Adversarial Datasets and Code Line-Level Feedback](http://arxiv.org/abs/2412.17754v1), 2024-12-23

(5) [Deliberation in Latent Space via Differentiable Cache Augmentation](http://arxiv.org/abs/2412.17747v1), 2024-12-23

(6) [RepoTransBench: A Real-World Benchmark for Repository-Level Code Translation](http://arxiv.org/abs/2412.17744v1), 2024-12-23

(7) [YuLan-Mini: An Open Data-efficient Language Model](http://arxiv.org/abs/2412.17743v1), 2024-12-23

(8) [Reasoning to Attend: Try to Understand How <SEG> Token Works](http://arxiv.org/abs/2412.17741v1), 2024-12-23

(9) [Chumor 2.0: Towards Benchmarking Chinese Humor Understanding](http://arxiv.org/abs/2412.17729v1), 2024-12-23

(10) [Knowledge Editing through Chain-of-Thought](http://arxiv.org/abs/2412.17727v1), 2024-12-23

(11) [Understanding the Logic of Direct Preference Alignment through Logic](http://arxiv.org/abs/2412.17696v1), 2024-12-23

(12) [RAGONITE: Iterative Retrieval on Induced Databases and Verbalized RDF for Conversational QA over KGs with RAG](http://arxiv.org/abs/2412.17690v1), 2024-12-23

(13) [Large Language Model Safety: A Holistic Survey](http://arxiv.org/abs/2412.17686v1), 2024-12-23

(14) [Generating Completions for Fragmented Broca's Aphasic Sentences Using Large Language Models](http://arxiv.org/abs/2412.17669v1), 2024-12-23

(15) [Detecting anxiety and depression in dialogues: a multi-label and explainable approach](http://arxiv.org/abs/2412.17651v1), 2024-12-23

(16) [SCBench: A Sports Commentary Benchmark for Video LLMs](http://arxiv.org/abs/2412.17637v1), 2024-12-23

(17) [Tracking the Feature Dynamics in LLM Training: A Mechanistic Study](http://arxiv.org/abs/2412.17626v1), 2024-12-23

(18) [Emerging Security Challenges of Large Language Models](http://arxiv.org/abs/2412.17614v1), 2024-12-23

(19) [Personalized Large Vision-Language Models](http://arxiv.org/abs/2412.17610v1), 2024-12-23

(20) [SBS Figures: Pre-training Figure QA from Stage-by-Stage Synthesized Images](http://arxiv.org/abs/2412.17606v1), 2024-12-23

(21) [EasyTime: Time Series Forecasting Made Easy](http://arxiv.org/abs/2412.17603v1), 2024-12-23

(22) [LiveIdeaBench: Evaluating LLMs' Scientific Creativity and Idea Generation with Minimal Context](http://arxiv.org/abs/2412.17596v1), 2024-12-23

(23) [Leveraging Memory Retrieval to Enhance LLM-based Generative Recommendation](http://arxiv.org/abs/2412.17593v1), 2024-12-23

(24) [HumanVBench: Exploring Human-Centric Video Understanding Capabilities of MLLMs with Synthetic Benchmark Data](http://arxiv.org/abs/2412.17574v1), 2024-12-23

(25) [GQSA: Group Quantization and Sparsity for Accelerating Large Language Model Inference](http://arxiv.org/abs/2412.17560v1), 2024-12-23

(26) [A Survey of Query Optimization in Large Language Models](http://arxiv.org/abs/2412.17558v1), 2024-12-23

(27) [Resource-Aware Arabic LLM Creation: Model Adaptation, Integration, and Multi-Domain Testing](http://arxiv.org/abs/2412.17548v1), 2024-12-23

(28) [Retention Score: Quantifying Jailbreak Risks for Vision Language Models](http://arxiv.org/abs/2412.17544v1), 2024-12-23

(29) [DiffusionAttacker: Diffusion-Driven Prompt Manipulation for LLM Jailbreak](http://arxiv.org/abs/2412.17522v1), 2024-12-23

(30) [DRT-o1: Optimized Deep Reasoning Translation via Long Chain-of-Thought](http://arxiv.org/abs/2412.17498v1), 2024-12-23

(31) [Is ChatGPT Massively Used by Students Nowadays? A Survey on the Use of Large Language Models such as ChatGPT in Educational Settings](http://arxiv.org/abs/2412.17486v1), 2024-12-23

(32) [Power- and Fragmentation-aware Online Scheduling for GPU Datacenters](http://arxiv.org/abs/2412.17484v1), 2024-12-23

(33) [A Silver Bullet or a Compromise for Full Attention? A Comprehensive Study of Gist Token-based Context Compression](http://arxiv.org/abs/2412.17483v1), 2024-12-23

(34) [A Survey on Multi-Generative Agent System: Recent Advances and New Frontiers](http://arxiv.org/abs/2412.17481v1), 2024-12-23

(35) [Applying LLM and Topic Modelling in Psychotherapeutic Contexts](http://arxiv.org/abs/2412.17449v1), 2024-12-23

(36) [Condor: A Code Discriminator Integrating General Semantics with Code Details](http://arxiv.org/abs/2412.17429v1), 2024-12-23

(37) [Measuring Contextual Informativeness in Child-Directed Text](http://arxiv.org/abs/2412.17427v1), 2024-12-23

(38) [Multimodal Preference Data Synthetic Alignment with Reward Model](http://arxiv.org/abs/2412.17417v1), 2024-12-23

(39) [VidCtx: Context-aware Video Question Answering with Image Models](http://arxiv.org/abs/2412.17415v1), 2024-12-23

(40) [Just What You Desire: Constrained Timeline Summarization with Self-Reflection for Enhanced Relevance](http://arxiv.org/abs/2412.17408v1), 2024-12-23

(41) [Towards Intrinsic Self-Correction Enhancement in Monte Carlo Tree Search Boosted Reasoning via Iterative Preference Learning](http://arxiv.org/abs/2412.17397v1), 2024-12-23

(42) [WarriorCoder: Learning from Expert Battles to Augment Code Large Language Models](http://arxiv.org/abs/2412.17395v1), 2024-12-23

(43) [Interweaving Memories of a Siamese Large Language Model](http://arxiv.org/abs/2412.17383v1), 2024-12-23

(44) [Boosting LLM via Learning from Data Iteratively and Selectively](http://arxiv.org/abs/2412.17365v1), 2024-12-23

(45) [MineAgent: Towards Remote-Sensing Mineral Exploration with Multimodal Large Language Models](http://arxiv.org/abs/2412.17339v1), 2024-12-23

(46) [A Dual-Perspective Metaphor Detection Framework Using Large Language Models](http://arxiv.org/abs/2412.17332v1), 2024-12-23

(47) [Assessing Human Editing Effort on LLM-Generated Texts via Compression-Based Edit Distance](http://arxiv.org/abs/2412.17321v1), 2024-12-23

(48) [CodeV: Issue Resolving with Visual Data](http://arxiv.org/abs/2412.17315v1), 2024-12-23

(49) [On the Feasibility of Vision-Language Models for Time-Series Classification](http://arxiv.org/abs/2412.17304v1), 2024-12-23

(50) [Prompting in the Wild: An Empirical Study of Prompt Evolution in Software Repositories](http://arxiv.org/abs/2412.17298v1), 2024-12-23

(51) [AV-EmoDialog: Chat with Audio-Visual Users Leveraging Emotional Cues](http://arxiv.org/abs/2412.17292v1), 2024-12-23

(52) [Multi-Modal Grounded Planning and Efficient Replanning For Learning Embodied Agents with A Few Examples](http://arxiv.org/abs/2412.17288v1), 2024-12-23

(53) [LLM4AD: A Platform for Algorithm Design with Large Language Model](http://arxiv.org/abs/2412.17287v1), 2024-12-23

(54) [LegalAgentBench: Evaluating LLM Agents in Legal Domain](http://arxiv.org/abs/2412.17259v1), 2024-12-23

(55) [Unlocking Cross-Lingual Sentiment Analysis through Emoji Interpretation: A Multimodal Generative AI Approach](http://arxiv.org/abs/2412.17255v1), 2024-12-23

(56) [SyNeg: LLM-Driven Synthetic Hard-Negatives for Dense Retrieval](http://arxiv.org/abs/2412.17250v1), 2024-12-23

(57) [EM-MIAs: Enhancing Membership Inference Attacks in Large Language Models through Ensemble Modeling](http://arxiv.org/abs/2412.17249v1), 2024-12-23

(58) [On the Generalization Ability of Machine-Generated Text Detectors](http://arxiv.org/abs/2412.17242v1), 2024-12-23

(59) [Brain-to-Text Benchmark '24: Lessons Learned](http://arxiv.org/abs/2412.17227v1), 2024-12-23

(60) [Better Think with Tables: Leveraging Tables to Enhance Large Language Model Comprehension](http://arxiv.org/abs/2412.17189v1), 2024-12-22

(61) [Enhancing Item Tokenization for Generative Recommendation through Self-Improvement](http://arxiv.org/abs/2412.17171v1), 2024-12-22

(62) [LLM-based relevance assessment still can't replace human relevance assessment](http://arxiv.org/abs/2412.17156v1), 2024-12-22

(63) [A Multi-AI Agent System for Autonomous Optimization of Agentic AI Solutions via Iterative Refinement and LLM-Driven Feedback Loops](http://arxiv.org/abs/2412.17149v1), 2024-12-22

(64) [LLM Agent for Fire Dynamics Simulations](http://arxiv.org/abs/2412.17146v1), 2024-12-22

(65) [Hate Speech Detection and Target Identification in Devanagari Languages via Parameter Efficient Fine-Tuning of LLMs](http://arxiv.org/abs/2412.17131v1), 2024-12-22

(66) [Lies, Damned Lies, and Distributional Language Statistics: Persuasion and Deception with Large Language Models](http://arxiv.org/abs/2412.17128v1), 2024-12-22

(67) [DreamOmni: Unified Image Generation and Editing](http://arxiv.org/abs/2412.17098v1), 2024-12-22

(68) [Analysis on LLMs Performance for Code Summarization](http://arxiv.org/abs/2412.17094v1), 2024-12-22

(69) [SAIL: Sample-Centric In-Context Learning for Document Information Extraction](http://arxiv.org/abs/2412.17092v1), 2024-12-22

(70) [SubstationAI: Multimodal Large Model-Based Approaches for Analyzing Substation Equipment Faults](http://arxiv.org/abs/2412.17077v1), 2024-12-22

(71) [The HalluRAG Dataset: Detecting Closed-Domain Hallucinations in RAG Applications Using an LLM's Internal States](http://arxiv.org/abs/2412.17056v1), 2024-12-22

(72) [DR-Encoder: Encode Low-rank Gradients with Random Prior for Large Language Models Differentially Privately](http://arxiv.org/abs/2412.17053v1), 2024-12-22

(73) [ViLBias: A Framework for Bias Detection using Linguistic and Visual Cues](http://arxiv.org/abs/2412.17052v1), 2024-12-22

(74) [Modular Conversational Agents for Surveys and Interviews](http://arxiv.org/abs/2412.17049v1), 2024-12-22

(75) [Why Do Speech Language Models Fail to Generate Semantically Coherent Outputs? A Modality Evolving Perspective](http://arxiv.org/abs/2412.17048v1), 2024-12-22

(76) [Shaping the Safety Boundaries: Understanding and Defending Against Jailbreaks in Large Language Models](http://arxiv.org/abs/2412.17034v1), 2024-12-22

(77) [MINTQA: A Multi-Hop Question Answering Benchmark for Evaluating LLMs on New and Tail Knowledge](http://arxiv.org/abs/2412.17032v1), 2024-12-22

(78) [FriendsQA: A New Large-Scale Deep Video Understanding Dataset with Fine-grained Topic Categorization for Story Videos](http://arxiv.org/abs/2412.17022v1), 2024-12-22

(79) [Robustness of Large Language Models Against Adversarial Attacks](http://arxiv.org/abs/2412.17011v1), 2024-12-22

(80) [LLM-Powered User Simulator for Recommender System](http://arxiv.org/abs/2412.16984v1), 2024-12-22

(81) [On Fusing ChatGPT and Ensemble Learning in Discon-tinuous Named Entity Recognition in Health Corpora](http://arxiv.org/abs/2412.16976v1), 2024-12-22

(82) [Cannot or Should Not? Automatic Analysis of Refusal Composition in IFT/RLHF Datasets and Refusal Behavior of Black-Box LLMs](http://arxiv.org/abs/2412.16974v1), 2024-12-22

(83) [System-2 Mathematical Reasoning via Enriched Instruction Tuning](http://arxiv.org/abs/2412.16964v1), 2024-12-22

(84) [Aristotle: Mastering Logical Reasoning with A Logic-Complete Decompose-Search-Resolve Framework](http://arxiv.org/abs/2412.16953v1), 2024-12-22

(85) [A Career Interview Dialogue System using Large Language Model-based Dynamic Slot Generation](http://arxiv.org/abs/2412.16943v1), 2024-12-22

(86) [Prompting Large Language Models with Rationale Heuristics for Knowledge-based Visual Question Answering](http://arxiv.org/abs/2412.16936v1), 2024-12-22

(87) [Towards a Unified Paradigm: Integrating Recommendation Systems as a New Language in Large Models](http://arxiv.org/abs/2412.16933v1), 2024-12-22

(88) [Enhancing Supply Chain Transparency in Emerging Economies Using Online Contents and LLMs](http://arxiv.org/abs/2412.16922v1), 2024-12-22

(89) [PsychAdapter: Adapting LLM Transformers to Reflect Traits, Personality and Mental Health](http://arxiv.org/abs/2412.16882v1), 2024-12-22

(90) [Online Preference-based Reinforcement Learning with Self-augmented Feedback from Large Language Model](http://arxiv.org/abs/2412.16878v1), 2024-12-22

(91) [Teaching LLMs to Refine with Tools](http://arxiv.org/abs/2412.16871v1), 2024-12-22

(92) [CoF: Coarse to Fine-Grained Image Understanding for Multi-modal Large Language Models](http://arxiv.org/abs/2412.16869v1), 2024-12-22

(93) [GME: Improving Universal Multimodal Retrieval by Multimodal LLMs](http://arxiv.org/abs/2412.16855v1), 2024-12-22

(94) [Sim911: Towards Effective and Equitable 9-1-1 Dispatcher Training with an LLM-Enabled Simulation](http://arxiv.org/abs/2412.16844v1), 2024-12-22

(95) [Ask-Before-Detection: Identifying and Mitigating Conformity Bias in LLM-Powered Error Detector for Math Word Problem Solutions](http://arxiv.org/abs/2412.16838v1), 2024-12-22

(96) [Online Learning from Strategic Human Feedback in LLM Fine-Tuning](http://arxiv.org/abs/2412.16834v1), 2024-12-22

(97) [KG4Diagnosis: A Hierarchical Multi-Agent LLM Framework with Knowledge Graph Enhancement for Medical Diagnosis](http://arxiv.org/abs/2412.16833v1), 2024-12-22

(98) [Visual Prompting with Iterative Refinement for Design Critique Generation](http://arxiv.org/abs/2412.16829v1), 2024-12-22

(99) [An Exploration of Pattern Mining with ChatGPT](http://dx.doi.org/10.1145/3698322.3698338), 2024-12-22
